package.json

```diff
@@ -28,9 +28,11 @@
     "@middy/http-error-handler": "^1.0.0-beta.6",
     "@middy/http-json-body-parser": "^1.0.0-beta.6",
     "@middy/validator": "^1.0.0-beta.6",
+    "ejs": "^3.0.1",
+    "mongoose": "^5.8.3",
+    "nodemailer": "^6.4.3",
     "@types/aws-lambda": "^8.10.45",
     "http-errors": "^1.7.3",
-    "mongoose": "^5.9.2",
```

Comment:
Can you rebase off master here so that it doesn't drop down our dependency version